airflow/models/taskinstance.py

```diff
@@ -1308,6 +1296,29 @@ def _run_execute_callback(self, context, task):
             self.log.error("Failed when executing execute callback")
             self.log.exception(exc)
 
+    def run_finished_callback(self) -> None:
```

Comment:
```suggestion
    def _run_finished_callback(self) -> None:
```

Since it should be "private"